### DISCUSSION TITLE
将Halo2.9进行备份后，重新安装Halo2.11后导入备份，编辑文章时出现问题

### DISCUSSION THREAD

**Discussion by duguyue8** [halo-dev/halo#5012] [Q&A]
点击进入文章的页面编辑就出现如下的错误：
![image.png](https://pic7.58cdn.com.cn/nowater/webim/big/n_v22ac312f9428a4d3fa841847e8ea5085f.png)
在原来写的文章修改后，点击保存时，新写的内容消失。

**Comment by JohnNiang** [comment]
确认一下 CDN 是否有缓存接口请求。
